pytest_plugins/skip_component_tests.py

```diff
@@ -0,0 +1,45 @@
+from robottelo.logging import collection_logger as logger
+
+skipped_components = ['Virt-whoConfigurePlugin']
```

Comment:
This could also be read from `pyproject.toml`

Reply:
I dont see its necessarily configurable because we are not frequently run/skip those tests in SatQE CI, its like set in stone that we will never run this test in SAT QE CI.

Reply:
@JacobCallahan I think this is not necessary now, we might implement this as an improvement later if people demands!